src/main/groovy/edu/oregonstate/mist/cardsapi/resources/CardsResource.groovy

```diff
@@ -1,37 +1,46 @@
 package edu.oregonstate.mist.cardsapi.resources
 
 import io.dropwizard.jersey.params.IntParam
-import io.dropwizard.auth.Auth
 import edu.oregonstate.mist.cardsapi.core.Card
-import edu.oregonstate.mist.api.AuthenticatedUser
-
 import edu.oregonstate.mist.cardsapi.db.CardDAO
+import edu.oregonstate.mist.cardsapi.db.CardFluent
 import edu.oregonstate.mist.api.Resource
 import edu.oregonstate.mist.api.jsonapi.ResourceObject
 import edu.oregonstate.mist.api.jsonapi.ResultObject
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
 
+import javax.ws.rs.DELETE
 import javax.annotation.security.PermitAll
 import javax.ws.rs.GET
 import javax.ws.rs.Path
 import javax.ws.rs.PathParam
 import javax.ws.rs.Produces
 import javax.ws.rs.core.Response
 import javax.ws.rs.core.MediaType
+import javax.ws.rs.QueryParam
+import javax.validation.Valid
+import com.google.common.base.Optional
+
+import org.skife.jdbi.v2.DBI
 
 // This will get a Card object from CardDAO and send responses for different endpoints
 
 @Path('/cards')
 @PermitAll
 @Produces(MediaType.APPLICATION_JSON)
 class CardsResource extends Resource {
-    Logger logger = LoggerFactory.getLogger(CardsResource.class)
 
     private final CardDAO cardDAO
+    private DBI dbi
+    CardFluent cardFluent = new CardFluent(dbi)
 
-    CardsResource(CardDAO cardDAO) {
+    List<String> validTypes = ["skill", "attack", "power", "status", "curse"]
+    List<String> validColors = ["red", "green", "blue", "colorless"]
+    List<String> validRarities = ["basic", "common", "uncommon", "rare"]
+    String regEx = '[a-zA-Z0-9 ."+-]*'
```

Comment:
Could you change the name of this variable or add a comment to indicate what its purpose is?